Source/Core/Common/ArmCPUDetect.cpp

```diff
@@ -60,6 +67,20 @@ void CPUInfo::Detect()
   Mode64bit = true;
   vendor = CPUVendor::ARM;
 
+#ifdef _WIN32
+  // Windows does not provide any mechanism for querying the system registers on ARMv8, unlike Linux
+  // which traps the register reads and emulates them in the kernel. There are environment variables
+  // containing some of the CPU-specific values, which we could use for a lookup table in the
+  // future. For now, assume all features are present as all known devices which are Windows-on-ARM
+  // compatible also support these extensions.
+  num_cores = 4;
```

Comment:
My understanding is that `_SC_NPROCESSORS_CONF` returns number of currently available hardware threads, so you could just use `std::thread::hardware_concurrency` here?